src/main/rx/rx.c

```diff
@@ -616,12 +616,11 @@ bool calculateRxChannelsAndUpdateFailsafe(timeUs_t currentTimeUs)
         processingRequired = !rxRuntimeConfig.rcProcessFrameFn(&rxRuntimeConfig);
     }
 
-    if (!rxDataReceived) {
+    if (!(rxDataReceived || (currentTimeUs >= rxUpdateAt))) {
```

Comment:
This looks a little confusing. Those brace around currentTimeUs >= rxUpdateAt are strange, but even without the brances i have problems understanding it :(
Maybe !rxDataReceived || currentTimeUs <= currentTimeUs  would be easier to understand

Reply:
@wind0r : `!rxDataReceived && currentTimeUs < rxUpdateAt` (or better  `!rxDataReceived && cmpTimeUs(currentTimeUs, rxUpdateAt) < 0`